src/libraries/System.IO.Compression.ZipFile/tests/ZipFile.Unix.cs

```diff
@@ -12,7 +12,8 @@ namespace System.IO.Compression.Tests
     public partial class ZipFile_Unix : ZipFileTestBase
     {
         [Fact]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/60581", TestPlatforms.iOS | TestPlatforms.tvOS)]
+        // Disabled on Mac for https://github.com/dotnet/runtime/issues/68293
```

Comment:
Should we add a second `ActiveIssue` attribute for OSX instead of reusing the existing one and having the comment?

Reply:
I didn't realize ActiveIssue was allowmultiple, but I see it is. I'll do that.

Reply:
done